app/adapters/whats_app_adapter/three_sixty_dialog_inbound.rb

```diff
@@ -19,20 +19,18 @@ def consume(organization, whats_app_payload)
       @quote_reply_message_id = message_payload.dig(:context, :id)
       @text = initialize_text
 
-      if ephemeral_data?
-        handle_ephemeral_data
-        return
-      end
+      return if ephemeral_data_handled?
 
       @message = initialize_message
-      @message.request = initialize_request
 
       @unsupported_content = initialize_unsupported_content
 
       files = initialize_file
       @message.files = files
+      @message.request = sender.active_request
 
-      yield(@message) if block_given?
```

Comment:
Not sure if the block was used somewhere..

Reply:
the `{ |message| message.contributor.reply(adapter) }` block was passed in, but it's been removed.